intern_program/intern_program_fall_2018_projects.md

```diff
@@ -34,3 +34,30 @@ This includes building a `.json` catalog of the existing datasets, integration a
 
 #### Skills Required
 Python programming, basic knowledge of html
+
+
+### Building and analyzing 3D virtual terrain models of lava domes from lab experiments and the field
+
+#### Supervisor
+Einat Lev, Lamont Associate Research Professor
+
+#### Description
+Lava domes form when highly viscous lava erupts from a volcano and piles up above the source vent. Domes are higly unstable and collapse frequently, forming dangerous currents of ash and debris. This summer we performed a series of experiments where we simulated lava dome emplacement using wax+clay slurries, and studied the influence of different parameters on the structure of the resulting dome. We collected lots of pictures of the wax domes, and now would like to use photogrammetry to create virtual 3D models of them. Then, the student will apply the same skills to constructing 3D terrain models of lava domes and flows we colelcted using drones in Indonesia, Ecuador, and Hawai'i. All the models will be analyzed for surface fractures and roughness using Matlab/Python image processing tools and QGIS/ArcGIS mapping tools. 
+
+All resulting models will be shared with the public via https://sketchfab.com/LDEO-volcanology and will accompany educational activities. 
+
+#### Skills Required
+Matlab, Python.  
```

Comment:
Can we clarify with this is Matlab AND Python vs. Matlab OR Python. (Most of the undergrad CS majors do not have any Matlab experience, so that may be harder to satisfy).

Also, both of these are very broad. Would be useful more specific on the skills. For example "image processing in python using scikit-image".